experimental/PropertyDDS/packages/property-shared-tree-interop/README.md

```diff
@@ -0,0 +1,56 @@
+# @fluid-experimental/property-shared-tree-interop
+
+This package contains tools and utilities, that should help application developers to migrate their projects
+from `PropertyDDS` to the new `SharedTree` DDS (see the
+[@fluid-experimental/tree2](https://github.com/microsoft/FluidFramework/blob/main/experimental/dds/tree2/README.md)).
+
+## Schema converter (runtime)
+
+A [`schema-converter`](./src/schemaConverter.ts) converts a `PropertyDDS` schema template into a [`TypedSchemaCollection`](https://github.com/microsoft/FluidFramework/blob/main/experimental/dds/tree2/src/feature-libraries/modular-schema/typedSchema/schemaBuilder.ts) in runtime, so that the resulting schema can be used by the `SharedTree`, for example:
```

Comment:
> in runtime

Should that be "at runtime" or "during runtime"?

Reply:
Done.